contrib/auctex/extensions.el

```diff
@@ -16,12 +16,9 @@
   "Initialize reftex"
 
   (add-hook 'LaTeX-mode-hook 'turn-on-reftex)
-  (setq reftex-plug-into-AUCTeX t)
+  (setq reftex-plug-into-AUCTeX auctex-reftex-plugin)
 
-  ;; not supported for now
-  ;; (setq spacemacs/key-binding-prefixes '(("mr" . "RefTeX")))
-
-  (evil-leader/set-key-for-mode 'LaTeX-mode
+  (evil-leader/set-key-for-mode 'latex-mode
```

Comment:
`LaTeX-mode` didn't work for me, but when I changed it to `latex-mode`, it started working again; hence why I made the change.